client/src/components/Header.tsx

```diff
@@ -38,13 +38,16 @@ const InnerBar = styled(Inner)`
 
 interface TParams {
   electionId: string
+  jurisdiction?: string
+  jurisdictionId?: string
 }
 
 const Header: React.FC<{}> = () => {
   const match: RouteComponentProps<TParams>['match'] | null = useRouteMatch(
-    '/election/:electionId'
+    '/election/:electionId/:jurisdiction?/:jurisdictionId?'
```

Comment:
this feels kinda tricky, since `jurisdiction` isn't really a parameter, but rather just the string `jurisdiction`. maybe add a comment or just do two separate matches, one for the election path and one for the jurisdiction path

Reply:
I hadn't even thought of doing two matches, that actually cleaned things up nicely. :slightly_smiling_face: